src/compas/datastructures/_mixins/filters.py

```diff
@@ -265,7 +265,7 @@ def edges_where_predicate(self, predicate, data=False):
             mesh.set_edge_attributes(random_edge, ['extra_attr1', 'extra_attr2'], [2, [3,5,9]])
 
 
-            for u, v, e_data in mesh.edges_where_predicate(lambda u, v, attr: attr['extra_attr1'] == 2, True):
+            for u, v, e_data in mesh.edges_where_predicate(lambda u, v, attr: attr['extra_attr1']==2, True):
```

Comment:
there the whitespace is correct. ony case where there is no whitespace is in the case of function arguments....

Reply:
ok